lib/c-api/src/deprecated/mod.rs

```diff
@@ -0,0 +1,180 @@
+//! # Wasmer Runtime C API
```

Comment:
for future reference I think `git mv` can avoid the delete/readd problem and will preserve history (I often forget to use it too)

Reply:
Same here :-p.